robloxmutexer.c

```diff
@@ -26,12 +26,11 @@ main() {
 
 	printf("roblox mutex locked\n");
 
-	for (;;) {
-		/*
-		 * you cannot do anything until you kill me;
-		 * by then, the mutex is yours.
-		 */
-	}
+	/*
+	 * you cannot do anything until you kill me;
+	 * by then, the mutex is yours.
+	 */
+	result = WaitForSingleObject(GetCurrentProcess(), INFINITE);
```

Comment:
```suggestion
	WaitForSingleObject(GetCurrentProcess(), INFINITE);
```
result is not actually used or cared about in this case.